src/common/components/Button/README.md

```diff
@@ -4,15 +4,15 @@ Buttons are used to invoke an event.
 
 ## Props
 
-| Name            | Required | Type          | Possible values                                                                     | Notes                                                            |
-| --------------- | -------- | ------------- | ----------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
-| `kind`          |          | string        | `'primary'`, `'secondary'`, `'secondaryNegative'`, `'success'`, `'warn'`, `'text'`, | Use `secondaryNegative` only if you have an off white background |
-| `size`          |          | string        | `'small'`, `'medium'`, `'large'`                                                    |                                                                  |
-| `icon`          |          | `JSX.Element` |                                                                                     | Icon _must_ contain a height in the SVG to show up.              |
-| `iconAlignment` |          | string        | `'start'`, `'end'`                                                                  | Defaults to `start`                                              |
-| `onClick`       |          | `() => void`  |                                                                                     |                                                                  |
-| `disabled`      |          | boolean       |                                                                                     |                                                                  |
-| `loading`       |          | boolean       |                                                                                     |                                                                  |
+| Name            | Required | Type          | Possible values                                                                                 | Notes                                                            |
+| --------------- | -------- | ------------- | ----------------------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
+| `kind`          |          | string        | `'primary'`, `'purple'`, `'secondary'`, `'secondaryNegative'`, `'success'`, `'warn'`, `'text'`, | Use `secondaryNegative` only if you have an off white background |
```

Comment:
this change doesn't correspond with the `ButtonKind` type below

Reply:
Yep, noticed it myself actually and commited a fix :p

Reply:
👍🏼😀